modules/waypoints_to_spline_commands.py

```diff
@@ -0,0 +1,58 @@
+"""
+Function to convert list of waypoints a list of spline waypoint dronekit commands.
+"""
+
+import dronekit
+
+from .common.kml.modules import location_ground
+
+
+MAVLINK_FRAME = dronekit.mavutil.mavlink.MAV_FRAME_GLOBAL_RELATIVE_ALT
+MAVLINK_COMMAND = dronekit.mavutil.mavlink.MAV_CMD_NAV_SPLINE_WAYPOINT
+
+
+def waypoints_to_spline_commands(waypoints: "list[location_ground.LocationGround]",
+                                 altitude: int) -> "tuple[bool, list[dronekit.Command] | None]":
+    """
+    Convert list of waypoints to a list of spline waypoint dronekit commands.
+
+    Spline waypoint dronekit commands fly to the target waypoint following a spline path then hover in place.
```

Comment:
Nit: In brackets can you mention that a spline path is a path that curves around the waypoint.